templates/softlayer.yml.erb

```diff
@@ -0,0 +1,123 @@
+---
```

Comment:
now that custom manifest path can be specified, softlayer.yml.erb can be kept in bosh-softlayer-cpi-release repo and pointed at via path.